vme/src/db_file.cpp

```diff
@@ -1173,11 +1180,8 @@ void bwrite_block(FILE *datafile, int length, void *buffer)
 int write_unit_string(CByteBuffer *pBuf, unit_data *u)
 {
     int i = 0;
-
-#ifdef DMSERVER
```

Comment:
Suppose I'm wondering about this line. Presumably it was there related to something with VMC... ?

Reply:
Yeah, I think they were to reduce compile times. Adding some of the json_helper stuff to mplex dragged in almost everything else. There is a lot of dependencies between translation units :-)